superset-frontend/src/views/CRUD/chart/ChartList.tsx

```diff
@@ -285,7 +285,7 @@ function ChartList(props: ChartListProps) {
           </a>
         ),
         Header: t('Modified by'),
-        accessor: 'last_saved_by',
+        accessor: 'last_saved_by.id',
```

Comment:
does this result in an alphabetical sort?

Reply:
hmm should probably change this to .lastname then?

Reply:
i'll do first name since the original logic was to use first name.